x-pack/legacy/plugins/security/index.js

```diff
@@ -173,13 +173,15 @@ export const security = (kibana) => new kibana.Plugin({
       return new savedObjects.SavedObjectsClient(callWithRequestRepository);
     });
 
-    savedObjects.addScopedSavedObjectsClientWrapperFactory(Number.MIN_SAFE_INTEGER, 'security', ({ client, request }) => {
```

Comment:
I think we still want to reorder the SOC wrappers, as it makes the [authorization check](https://github.com/elastic/kibana/pull/42452/files#diff-a199cbb2b9f7afc4d6663274f8350bddR113-R122) more straightforward.

When re-ordered, the security plugin can always use the `namespace` option passed in for each operation. When the spaces SOC wrapper is being used, it will supply the `namespace`. When it isn't being used, then the consumer will be able to specify their own `namespace`.

When the wrappers aren't reordered, then the security wrapper needs to do the `options.hasOwnProperty('namespace')` vs `typeof options.namespace === 'undefined;` dance, which we want to avoid if we can.